### PR TITLE
 crDroid 11.0 :Initial support panther, cheetah, lynx

### PR DESCRIPTION
**Pull request by dominator-prime**

### CODE REVIEW

File: cheetah.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 7 Pro",
+			"filename": "crDroidAndroid-15.0-20241106-cheetah-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/cheetah/11.x/crDroidAndroid-15.0-20241106-cheetah-v11.0-BETA.zip/download",
+			"timestamp": 1730873149,
+			"md5": "8dcd2afec13c774594eef5a5562339c0",
+			"sha256": "de3e66455d6e47259349df158f40a3a0b34117dfb5ec7cba09951beb1165eb6d",
+			"size": 1584139211,
+			"version": "11.0",
+			"buildtype": "Weekly",
```
Comment (gwolf2u) [inline cheetah.json]: BETA

File: cheetah.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-cheetah-panther-14-0-crdroidandroid-v10-x.4648315/",
```
Comment (gwolf2u) [inline cheetah.json:15]: update forum template

File: cheetah.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/cheetah/11.x/recovery/",
+			"paypal": "https://www.paypal.me/vikasyaduvanshii",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_pantah",
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline cheetah.json:25]: missing branch

File: cheetah_changelog.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type:  Weekly
```
Comment (gwolf2u) [inline cheetah_changelog.txt]: BETA

File: lynx.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 7a",
+			"filename": "crDroidAndroid-15.0-20241106-lynx-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lynx/11.x/crDroidAndroid-15.0-20241106-lynx-v11.0-BETA.zip/download",
+			"timestamp": 1730886923,
+			"md5": "ac172d6c5f14f3b314c627cfa65eba56",
+			"sha256": "08064f559c005a4965d5225d6612e82ac63b651c96269016e2349d006a459446",
+			"size": 1591510367,
+			"version": "11.0",
+			"buildtype": "Weekly",
```
Comment (gwolf2u) [inline lynx.json]: BETA

File: lynx.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-14-lynx-crdroid-10-25-12-2023-lynx.4647462/",
```
Comment (gwolf2u) [inline lynx.json:15]: update forum template

File: lynx.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/lynx/11.x/recovery/",
+			"paypal": "https://www.paypal.me/vikasyaduvanshii",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_lynx",
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline lynx.json:25]: missing branch

File: lynx_changelog.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type:  Weekly
```
Comment (gwolf2u) [inline lynx_changelog.txt]: BETA

File: panther.json [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 7",
+			"filename": "crDroidAndroid-15.0-20241106-panther-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/panther/11.x/crDroidAndroid-15.0-20241106-panther-v11.0-BETA.zip/download",
+			"timestamp": 1730880151,
+			"md5": "4a26b29d392005fa9e256e6ec11c0752",
+			"sha256": "74187369cc84ff1656aff5725233fbe99602419ab735ddc4e41dac0fb0f8c955",
+			"size": 1581910983,
+			"version": "11.0",
+			"buildtype": "Weekly",
```
Comment (gwolf2u) [inline panther.json]: BETA

File: panther.json [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-cheetah-panther-14-0-crdroidandroid-v10-x.4648315/",
```
Comment (gwolf2u) [inline panther.json:15]: update forum template

File: panther.json [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/panther//11.x/recovery/",
+			"paypal": "https://www.paypal.me/vikasyaduvanshii",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_pantah",
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline panther.json:25]: missing branch

File: panther_changelog.txt [hunk 12]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type:  Weekly
```
Comment (gwolf2u) [inline panther_changelog.txt]: BETA

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: panther.json (hunk 11 above)
Comment (dominator-prime) [reply panther.json:25]: now we are using directly from aosp-pixel source   ask Pranav for clearing your doubts

**Pull request closed without merging**